frontend/src/static/js/v_summary.js

```diff
@@ -554,6 +567,14 @@ window.vSummary = {
 
     updateTmpFilterUntilDate(event) {
       const until = event.target.value;
+      this.hasModifiedUntilDate = true;
+
+      if (!this.isSafariBrowser) {
+        this.tmpFilterUntilDate = until;
+        event.target.value = this.filterUntilDate;
```

Comment:
why is it that we need to set the `event.target.value`?

Reply:
Because clearing the date fields using the HTML5 date picker button will clear the `event.target.value`. Without setting back `event.target.value`, clearing the date field using the button will result to `dd/MM/yyyy` instead of the filterSinceDate.